src/message_log.rs

```diff
@@ -268,7 +269,7 @@ impl PbftLog {
     }
 
     /// Obtain messages from the log that match a given type, sequence number, and view
-    pub fn get_messages_of_type(
+    pub fn get_messages_of_type_seq_view(
```

Comment:
It's a shame we can't do optional keyword parameters here, but can we just make `view` an `Option<u64>` and call this `filter_messages` or somesuch and have it cover both cases? If we wanted to get fancy, we could make `view` an `Into<Option<u64>>`, which would let us keep the calls that include the view parameter the same.

Reply:
It is a shame, but I personally prefer to leave it as-is; because these are relatively simple and straightforward functions, I'd rather make two easy-to-use functions than one more robust function that requires extra stuff (e.g. wrapping an argument in an `Option`)